### ISSUE TITLE
libdarknet.so

### ISSUE THREAD

**Issue by Goru1890** [daisukekobayashi/darknet-docker#6]
I'm very new in the world of containers and docker. 

I'm planning to use docker to contain a qt applications that uses the darknet api on an linux enviroment, is darknet compiled with the lib option in this project? I need that the application works on windows, my problem is that the darknet api doesn't work very well on windows.

Would it be possible to call the include files from the darknet API from the docker to an windows environment?

**Comment by daisukekobayashi** [comment]
No, Darknet isn’t compiled with the lib option in this project.
Did you check [yolo_cpp_dll](https://github.com/AlexeyAB/darknet/tree/master/build/darknet)  in [AlexyAB/darknet](https://github.com/AlexeyAB/darknet)? [AlexyAB/darknet](https://github.com/AlexeyAB/darknet) supports windows, so I recommend that you use yolo_cpp_dll. I have used the DLL before, and there wasn’t no problem.

You can’t call darknet API from the docker to an windows environment directly. If you want to do like that, first you have to make service like rpc or http and call the service in windows.

**Comment by Goru1890** [comment]
Thank you for your response.

Yes, darknet "works" on windows.

But I can't handle error like I do in linux. 
![image](https://user-images.githubusercontent.com/35274844/110912664-7f1b1d00-8314-11eb-8ea7-0aac72ec9d4a.png)

The call never ends when I have a cuda error on windows.
![image](https://user-images.githubusercontent.com/35274844/110912826-ad006180-8314-11eb-90c5-1788a5bdf82e.png)

That's why I'm thinking on using a docker with linux on a windows enviroment, but if there is no way to call the darknet api I'm out of options.

**Comment by gmontamat** [comment]
Hey @Goru1890 check out my fork here: https://github.com/gmontamat/darknet-docker
I also needed `libdarknet.so` on my projects so I modified the Dockerfiles on this repo. It's still a WIP but I verified the lib is generated successfully.
